lib/nerves_hub_link/client.ex

```diff
@@ -127,6 +138,22 @@ defmodule NervesHubLink.Client do
     end
   end
 
+  @doc """
+  Spawns a reboot. This allows the process that
+  calls this function to return before the reboot actually
+  starts.
+  """
+  def spawn_reboot do
```

Comment:
This needs specs and `()` since it's a public function and we've gone with the all-functions-have-parens-style.